### DISCUSSION TITLE
Linux Workstations and Monitoring Drives

### DISCUSSION THREAD

**Discussion by iSparkySystems** [amidaware/tacticalrmm#1043] [Q&A]
Hello Tactical RMM

When Creating a linux Workstation Agent receive the error missing code signing token - Not to sure what to do here, updated to latest version an hour or so ago.

Still frustrated by Drive Space Usage for Windows - No Drives listed in the combo box - same as above anything I must enable perhaps?

Thanx for your time.

**Comment by dinger1986** [comment]
Are you using code signing? Or are you compiling it? Support for multiple issues is easiest on discord.
